PostHog/Autocapture/PostHogAutocaptureEventTracker.swift

```diff
@@ -492,7 +524,7 @@
         return true
     }
 
-    // TODO: Filter out or obfuscsate strings that look like sensitive data
+    // TODO: Filter out or obfuscate strings that look like sensitive data
```

Comment:
an alternative for now is to use the `propertiesSanitizer`, you can mention this in the docs, but its a manoel (manual) job :D